### ISSUE TITLE
add a default menu bar on created pages

### ISSUE THREAD

**Issue by kurt1027** [HansSchouten/Laravel-Pagebuilder#106]
Hi sir @HansSchouten I successfully created a page but I want to add a default menu bar on these pages my problem is that the menu bar I want to add has some laravel functions included for example if the user is authenticated it will not show the login and signup but instead will show the name of the user since it's not a blade file the laravel functions will not work is there anyway to add the menu bar with laravel functions?

**Comment by HansSchouten** [comment]
Yes you can create a PHP block (use view.php instead of view.html) and then use any Laravel methods you want.

**Comment by kurt1027** [comment]
> Yes you can create a PHP block (use view.php instead of view.html) and then use any Laravel methods you want.

but laravel methods won't work if it's not a blade file sir

**Comment by HansSchouten** [comment]
Blade syntax won't work, but you can for example do <?= route('home') ?> and it will output an URL like expected.
